layouts/page/guideline.html

```diff
@@ -0,0 +1,204 @@
+                    <div class="bg-body-secondary mb-2 border-0 px-3 py-3" style="border-radius: 15px">
+                        <!--image-->
+                        <div class="d-flex justify-content-center mb-3">
+                            <img src="{{ absURL .icon }}" alt="{{ .department }} Icon" class="img-fluid"/>
+                        </div>
+
+                        <!--dropdown button-->
+                        <div class="dropdown d-block text-center">
+                            <button 
+                                style="border-radius: 10px; width: 100%"
+                                class="dropdown-toggle py-2 px-3 text-primary text-center border-0 fw-bold text-wrap"
+                                type="button"
+                                data-bs-toggle="dropdown"
+                                data-bs-display="static"
+                                data-bs-auto-close="outside"
+                                aria-expanded="false"
+                            >
+                                {{ .department -}}&nbsp;
+                            </button>
+                            <!--dropdown menu (mission & general tasks)-->
+                            <div class="dropdown-menu mt-2 bg-secondary px-3 pt-3" style="border-radius: 20px; border: none;">
+                                <div>
+                                    <span class="text-primary fw-bold">Mission</span>
+                                    <ul>
+                                        {{ range .mission }}
+                                            <li class="px-2 justify-content-center text-primary">{{ . }}</li>
+                                        {{ end }}
+                                    </ul>
+                                </div>
+                                <div>
+                                    <span class="text-primary fw-bold">General Task</span>
+                                    <ul>
+                                        {{ range .generalTask }}
+                                            <li class="px-2 justify-content-center">
+                                                {{ . }}
+                                            </li>
+                                        {{ end }}
+                                    </ul>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            {{ end }}
+        </div>
+    </section>
+
+    <section class="bg-body-secondary">
+        <div class="container" style="padding: 4em 1em;">
+            <div class="slider position-relative w-100">
+                <div class="slider-inner py-3 px-0 d-flex flex-nowrap position-relative">
+                    {{ range .Site.Params.joinUsImages }}
+                        <div class="slider-item d-block">
+                            <div class="box  mx-2" style='--bg: url("{{ absURL . }}");'></div>
```

Comment:
```suggestion
                            <div class="box rounded-1 mx-2" style='--bg: url("{{ absURL . }}");'></div>
```
in replacement of css above